src/main/webapp/resources/search/js/search.js

```
@@ -40,70 +40,96 @@ const state = {
+            if (trip !== "RT" && trip !== "OW") return;
+
+            handleTripChange(trip);
+
+            // 인디케이터 이동
+            tripSelector.setAttribute("data-trip", trip);
+
+            // active 클래스 토글
+            tripBtns.forEach(t => t.classList.toggle("active", t === btn));
+
+            syncTripUI();
+            syncDateSummary();
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)